pkg/exec/exec.go

```diff
@@ -0,0 +1,40 @@
+package exec
```

Comment:
I have no disagreement with the organizational intent here. My hesitation is that it doesn't take much to create an import cycle, and I feel like the more packages there are in play, the easier it becomes to encounter those situations.